Classes/Outlet/OutletArgument.php

```diff
@@ -0,0 +1,240 @@
+<?php
+namespace FluidTYPO3\Flux\Outlet;
+
+/*
+ * This file is part of the FluidTYPO3/Flux project under GPLv2 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+
+use TYPO3\CMS\Extbase\Error\Result;
+use TYPO3\CMS\Extbase\Mvc\Controller\MvcPropertyMappingConfiguration;
+use TYPO3\CMS\Extbase\Property\Exception\TargetNotFoundException;
+use TYPO3\CMS\Extbase\Property\PropertyMapper;
+use TYPO3\CMS\Extbase\Utility\TypeHandlingUtility;
+use TYPO3\CMS\Extbase\Validation\Exception\NoSuchValidatorException;
+use TYPO3\CMS\Extbase\Validation\Validator\ConjunctionValidator;
+use TYPO3\CMS\Extbase\Validation\Validator\ValidatorInterface;
+use TYPO3\CMS\Extbase\Validation\ValidatorResolver;
+use \TYPO3\CMS\Extbase\Mvc\Controller\Argument as ControllerArgument;
+use \TYPO3\CMS\Extbase\Property\Exception as PropertyException;
+
+/**
+ *
+ * @api
+ */
+class OutletArgument
```

Comment:
Has this class become more or less a duplicate of the `Argument` class used by controllers? Could we use that instead - or save a lot of duplication by subclassing it?